packages/oid4vci-demo-frontend/src/configs/banqup-account.json

```diff
@@ -1,10 +1,11 @@
 {
   "general": {
-    "credentialName": "Bank account credential",
-    "issueCredentialType": "BankAccount",
-    "baseUrl": "http://192.168.2.90:5000/bank",
+    "credentialName": "Account credential",
+    "issueCredentialType": "Account",
+    "oid4vpAgentBaseUrl": "https://account.banqup.demo.sphereon.com/agent",
+    "oid4vciAgentBaseUrl": "https://account.banqup.demo.sphereon.com/agent",
     "verifierUrl": "https://sphereon.com",
-    "verifierUrlCaption": "Bank credential",
+    "verifierUrlCaption": "Account credential",
```

Comment:
this needs to be a more general caption for the issuer. we have multiple credentials